modules/installation-osp-running-installer.adoc

```diff
@@ -0,0 +1,67 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+
+[id="installation-osp-running-installer_{context}"]
+= Running the {product-title} installer
```

Comment:
Probably a duplicate at this point. Incorporate into existing module and delete if so.